src/devices_models/devices/storage.jl

```diff
@@ -40,6 +40,20 @@ get_variable_initial_value(::EnergyVariable, d::PSY.Storage, ::AbstractStorageFo
 
 get_variable_binary(::ReserveVariable, ::Type{<:PSY.Storage}, ::AbstractStorageFormulation) = true
 
+get_target_multiplier(v::PSY.BatteryEMS) = PSY.get_rating(v)
```

Comment:
Why is this method here? This is not a model building function, it seems to just forward a method from PSY.

Reply:
I think its just stale code from previous implementation, I'll clean it up and also review the PR again to find anything like this

Reply:
I could move the `energy_target_constraint!` in the device file and use PSY method but that would case code repetition. as the multiplier for both devices is different 
https://github.com/NREL-SIIP/PowerSimulations.jl/blob/1ebf8edb897c634234cd555e1b8056e0a8d965a9/src/devices_models/devices/storage.jl#L43
https://github.com/NREL-SIIP/PowerSimulations.jl/blob/1ebf8edb897c634234cd555e1b8056e0a8d965a9/src/devices_models/devices/hydro_generation.jl#L103

Reply:
Some code repetition is preferable to make code more readable, we prefer dispatch at the construct device level. That's a tradeoff to make it easier to track what the models are doing.

Reply:
ok, I'll move this to the device file

Reply:
@sourabhdalvi did you implement this change?

Reply:
I did, just forgot to remove it